app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if current_user %>
```

Comment:
maybe we can put these on the layout (or a navbar)

Reply:
We have a ticket for the navbar, this was just as a way to test in the meantime, I'd rather have a clear view of where we are going css-wise